src/js/view/controls/components/chapters.mixin.js

```diff
@@ -4,28 +4,29 @@ define([
     'parsers/captions/srt',
 ], function(_, utils, srt) {
 
-    function Cue(time, text) {
-        this.time = time;
-        this.text = text;
-        this.el = document.createElement('div');
-        this.el.className = 'jw-cue jw-reset';
-    }
+    class Cue {
+
```

Comment:
@johnBartos check out https://github.com/eslint/eslint/issues/5949

I'm happy with this as-is.